cmd/printers/table/collector.go

```diff
@@ -182,8 +183,17 @@ func (r *ResourceStateCollector) processEvent(ev event.Event) error {
 // update for a resource.
 func (r *ResourceStateCollector) processStatusEvent(e event.StatusEvent) {
 	if e.Type == event.StatusEventResourceUpdate {
+		klog.V(7).Infoln("processing status event")
 		resource := e.Resource
-		previous := r.resourceInfos[resource.Identifier]
+		if resource == nil {
+			klog.V(4).Infoln("status event missing Resource field; no processing")
+			return
+		}
+		previous, found := r.resourceInfos[resource.Identifier]
+		if !found {
```

Comment:
Similar to above, but this seems like a situation that is a bug. If I remember correctly the `resourceInfos` map should be pre-populated with all resources which we are interested in. So if we get an event about a different resource, something must be wrong elsewhere.

Reply:
This probably happens when a `PersistentVolume` status hangs, but I haven't been able to reproduce this yet. Please have a look at the connected issue, which points out this `previous` is nil and not found.